tests/Client/UpdaterTest.php

```diff
@@ -1243,12 +1196,27 @@ public function testUpdateRefresh(): void
         $updater->refresh(true);
     }
 
+    public function providerUnsupportedRepo(): array
```

Comment:
Even though there's only one test case here, it needs to be moved to a data provider so that the subclasses can override the details as needed.